ui/shared/utils/constants.js

```diff
@@ -41,9 +41,13 @@ export const GENOME_VERSION_DISPLAY_LOOKUP = {
 
 // PROJECT FIELDS
 
+export const FILE_FIELD_NAME = 'uploadedFile'
+
+export const PROJECT_DESC_FIELDS = { name: 'description', label: 'Project Description', placeholder: 'Description' }
```

Comment:
Using plural for `FIELDS` is confusing because this is only one field, so I would rename it to `PROJECT_DESCRIPTION_FIELD`